codebuild/bin/s2n_codebuild.sh

```diff
@@ -65,6 +65,31 @@ if [[ -n "$S2N_NO_PQ" ]]; then
     CMAKE_PQ_OPTION="S2N_NO_PQ=True"
 fi
 
+setup_apache_server() {
+    # Start the apache server if the list of tests isn't defined, meaning all tests
+    # are to be run, or if the renegotiate test is included in the list of tests.
+    if [[ -z $TOX_TEST_NAME ]] || [[ "${TOX_TEST_NAME}" == *"test_renegotiate"* ]]; then
+        source codebuild/bin/s2n_apache2.sh
```

Comment:
Oh actually one more comment: why do you use $TOX_TEST_NAME here, but ${INTEGV2_TEST} on line 90? Is INTEGV2_TEST not used for the standard integrationv2 tests?

Reply:
For the standard integ tests, both variables should work in CI. Codebuild sets INTEGV2_TEST, which is set to TOX_TEST_NAME in the buildspec. However, for external users running our integ tests, our documentation says to set TOX_TEST_NAME, so this seems correct to check.

For criterion, it seems TOX_TEST_NAME isn't set until later, when calling make. Although, I think this line isn't being used currently, and it's removed in https://github.com/aws/s2n-tls/pull/3534 in favor of separate criterion buildspecs/scripts. To make this PR standalone, though, I added a test_names input to `setup_apache_server`.

Reply:
I actually think it makes more sense to just remove this here, and then there won't be conflicts when merging the criterion PR. 6ceadad96db7f3bb38fc0c65acb172976973362c